src/schema/interfaces.ts

```diff
@@ -16,6 +16,14 @@ export interface UserOutput {
   birthDate: string;
 }
 
+export interface UserList {
+  users: [UserOutput];
```

Comment:
Esse tipo quer dizer que esse campo vai reotrnar **um array de 1 elemento apenas**. Se o intuito é ser um array de UserOutput, é melhor assim:

```suggestion
  users: UserOutput[];
```